test/test-pages/gcba/sdk-controller.ts

```diff
@@ -0,0 +1,307 @@
+/* global MediaStream */
+import GenesysCloudWebrtSdk, { ConversationUpdate, IExtendedMediaSession, ISdkConfig, ISdkConversationUpdateEvent, IStoredConversationState, SdkError, SessionTypes } from '../../../src';
+import { getInputValue, writeToLog } from './utils';
+
+let webrtcSdk: GenesysCloudWebrtSdk;
+let pendingSessions = [];
+
+let conversationUpdatesToRender: {[conversationId: string]: IExtendedMediaSession} = { };
+
+export async function initWebrtcSDK (environmentData) {
+  const jwt = getInputValue('jwt-input');
+  if (!jwt) {
+    window.alert('You need a jwt token');
+    throw new Error('Not Authenticated');
+  }
+
+  const options: ISdkConfig = {
+    jwt,
+    originAppId: 'gcba-demo',
+    originAppName: 'gcba-demo',
+    autoAcceptPendingScreenRecordingRequests: true
+  }
+
+  options.environment = environmentData.uri;
+  options.logLevel = 'info';
+  // for sumo debugging
+  // options.optOutOfTelemetry = true;
+
+  const SDK = GenesysCloudWebrtSdk;
+  (window as any).SDK = SDK;
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9596369)
```suggestion
  (window as unknown).SDK = SDK;
```